README.rst

```diff
@@ -1,41 +1,230 @@
+The multi-vector simulator (MVS) allows the evaluation of local sector-coupled energy systems that include the energy carriers electricity, heat and/or gas.
+The MVS has three main features:
 
-- Analysis of an energy system model, which can be defined from csv or json files, including its
- costs and performance parameters.
- - Near-future investments into power generation and storage assets can be optimized aiming at
-  least-cost supply of electricity and heat.
- - Future energy supply scenarios that integrate emerging technologies helping to meet sustainability goals and decrease adverse climate effects can be evaluated, e.g. through high renewable energy shares or sector-coupling technologies.
+-  Analysis of an energy system model, which can be defined from csv or json files, including its costs and performance parameters.
+-  Near-future investments into power generation and storage assets can be optimized aiming at least-cost supply of electricity and heat.
+-  Future energy supply scenarios that integrate emerging technologies helping to meet sustainability goals and decrease adverse climate effects can be evaluated, e.g. through high renewable energy shares or sector-coupling technologies.
 
-The tool is being developed within the scope of the H2020 project E-LAND (Integrated multi-vector management system for Energy isLANDs, project homepage [HERE](https://elandh2020.eu/)). A graphical user interface for the MVS will be integrated.
+The tool is being developed within the scope of the H2020 project E-LAND (Integrated multi-vector management system for
+Energy isLANDs, project homepage `HERE <https://elandh2020.eu/>`__).
+A graphical user interface for the MVS will be integrated.
 
-*Latest release*
-Check the [latest release](https://github.com/rl-institut/multi-vector-simulator/releases/latest). Please check the [CHANGELOG.md](https://github.com/rl-institut/multi-vector-simulator/blob/master/CHANGELOG.md) for past updates and changes.
+*Latest release*: Check the `latest release <https://github.com/rl-institut/multi-vector-simulator/releases/latest>`__.
+Please check the `CHANGELOG.md <https://github.com/rl-institut/multi-vector-simulator/blob/master/CHANGELOG.md>`__ for past updates and changes.
 
-*Disclaimer*
-As the MVS is still under development, changes might still occur in the code as well as code
- structure. If you want to try the MVS, please make sure to check this project regularly.
+*Disclaimer*: As the MVS is still under development, changes might still occur in the code as well as code structure.
+If you want to try the MVS, please make sure to check this project regularly.
 
-For advanced programmers: You can also use the `dev` branch that includes the latest updates and
- changes. You find the changelog [HERE](https://github.com/rl-institut/multi-vector-simulator/blob/dev/CHANGELOG.md).
+For advanced programmers: You can also use the ``dev`` branch that includes the latest updates and changes.
```

Comment:
Are two ` needed? I always use one...

Reply:
One tutorial recommended it. That's why I used that.